roadmap.md

```diff
@@ -0,0 +1,60 @@
+creating new issues in the repository. When creating an issue, choose the template that best fits the issue you are about to create.  
+
+# Communicate
+- Use `@bf2fc6cc711aee1a0c2a/factorized-fleet` to alert the team about the issue. 
+- Use appropriate labels during the creation or the triaging of the issue. Labels help to categorise issues thus improving their discoverbility. Check out the existing labels on - https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/labels.  
+- Assign collaborators to issues to communicate responsibility.
+
+>NOTE: You can always reach out to the wider team via the available [communication channels](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/wiki/How-to-communicate-in-the-project) 
+
+# Assigning issue to a Sprint
+
+A Sprint is defined as an [iteration field](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/best-practices-for-managing-projects#use-different-field-types). An issue is assigned to a Sprint during Sprint planning.
+
+
+# View the Roadmap
+The roadmap is defined in the [project view](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) 
+The project view contains all the issues that have been triaged and potentially assigned to a Sprint.   
+
+The view is the source of truth and they are kept up to date to showing our roadmap.
+
+## How to add an issue to the Roadmap
+
+To add an issue to the Roadmap, go the [project](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) and click on the `Add Item` button (displayed down the board). 
+Github will propose a search input, where you can search for issues and add them to the board.
+
+## How to assign an issue to the Sprint
+
+Go to the [Project table view](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5/views/4?layout=table) which contains the `Sprint` column. 
+Look for the issue you are interested in. Then on the Sprint column, you could select a Sprint from the drop down list and assign it to the issue.
+
+## Filtering
+
+On the project you could filter by Label, Sprint, assignee, Status etc. See the [available filtering options](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/filtering-cards-on-a-project-board) 
+
+### Show only Epic
+
+To show Epics, use the filter `label: kind/epic`. This will show only larger goals issue.
+
+### Issue in a Sprint
+
+To show issues that are being progressed in a Spring `X`, use the filter `sprint:X` (where `X` is Sprint value).
+To show all issues in the current Sprint use the `sprint:@current` filter.
+
+# Usage of Milestone
+Milestones will be used as rough outlines of work assigned to a quarter. A milestone can be assigned to a smaller issue or an epic.
+Check out the available milestones - https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/milestones.
+
+# Automation
```

Comment:
This feels out of place. I would either remove it or caveat it in the sentence to explicitly call out that we want to introduce automation.

Reply:
Yes, this is very much a TODO section. I am okay to remove it and it will be added when we have automation in place.